requirements/requirements.in

```diff
@@ -1,6 +1,7 @@
 Click >= 7.0
 grpcio
 Jinja2 >= 2.10
+packaging
```

Comment:
oh I missed one thing, I don't think this is the right place for this requirement as we don't need it at runtime.

Since it's only needed at build time, it should only be in the `requires` section of `pyproject.toml`. And it seems like you've already added it there so I believe you can just remove this line

(this should also have an agreeable side-effect of removing all the dependency version changes from the  diff)